service/src/main/java/tds/assessment/repositories/impl/ItemMeasurementQueryRepositoryImpl.java

```diff
@@ -0,0 +1,65 @@
+package tds.assessment.repositories.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.jdbc.core.namedparam.SqlParameterSource;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+import tds.assessment.ItemMeasurement;
+import tds.assessment.repositories.ItemMeasurementQueryRepository;
+
+@Repository
+public class ItemMeasurementQueryRepositoryImpl implements ItemMeasurementQueryRepository {
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public ItemMeasurementQueryRepositoryImpl(final NamedParameterJdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public List<ItemMeasurement> findItemMeasurements(final String segmentKey, final String assessmentKey) {
+        SqlParameterSource parameters = new MapSqlParameterSource("segmentKey", segmentKey)
+            .addValue("assessmentKey", assessmentKey);
+
+        String SQL = "SELECT \n" +
+            "   coalesce(S.TestPosition, 1) as segmentPosition, \n" +
```

Comment:
I am porting legacy queries.  At this point in time I'd rather not shift around logic and just leave the queries as is unless we find performance issues.